app/code/community/Bolt/Boltpay/Helper/Data.php

```diff
@@ -89,4 +98,75 @@ public function canUseForCountry($country)
 
         return true;
     }
+
+    /**
+     * Get config value
+     *
+     * @return bool
+     */
+    public function isNeedAddButtonToMiniCart()
```

Comment:
Do we need this config? Can we always replace minicart if it presents? We can add config later if there is merchant who use minicart but don't want Bolt there.

Reply:
I think we should add it because every merchant has own classes in the MiniCart buttons.

Reply:
ok, we will replace every button if it present.